bulk-model-sync-gradle/src/test/kotlin/org/modelix/model/sync/bulk/gradle/MpsRunnerConfigurationTest.kt

```diff
@@ -0,0 +1,95 @@
+package org.modelix.model.sync.bulk.gradle
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.modelix.model.sync.bulk.gradle.config.LocalTarget
+import org.modelix.model.sync.bulk.gradle.config.ServerSource
+import org.modelix.model.sync.bulk.gradle.config.SyncDirection
+import java.io.File
+
+class MpsRunnerConfigurationTest {
+
+    private val jsonDir = File("/jsonDir")
+    private val classPathElements = emptySet<File>()
+
+    @Test
+    fun `build configuration for local target with a server source`() {
+        val serverSource = ServerSource()
+        val localTarget = LocalTarget(repositoryDir = File("/repositoryDir"))
+        val syncDirection = SyncDirection("syncDirection", serverSource, localTarget)
+
+        val config = buildMpsRunConfigurationForLocalTarget(syncDirection, classPathElements, jsonDir)
+        val jvmArgs = config.jvmArgs
+
+        val expectedJvmArgs = listOf(
+            "-Dmodelix.mps.model.sync.bulk.input.path=/jsonDir",
+            "-Dmodelix.mps.model.sync.bulk.input.modules=",
+            "-Dmodelix.mps.model.sync.bulk.input.modules.prefixes=",
+            "-Dmodelix.mps.model.sync.bulk.repo.path=/repositoryDir",
+            "-Dmodelix.mps.model.sync.bulk.input.continueOnError=false",
+            "-Xmx2g",
+        )
+
+        assertEquals(expectedJvmArgs, jvmArgs)
```

Comment:
Maybe use set comparison because the argument order probably doesn't matter?

Reply:
Yeah, that would be cleaner.
I changed it to Kotest to use the appropriate assertions.